docs/user_guide/hopsworks/kafka.rst

```diff
@@ -185,3 +165,81 @@ finally the third is the custom ACL we created before.
     :figclass: align-center
 
     Kafka topic details
+
+**Upload the compiled sample application and use it to create Spark jobs on Hopsworks**
+
+* Step 1: Upload the jar file from `hops-examples/spark/target/` to a dataset. The jar is named: `hops-examples-spark-X.Y.Z-SNAPSHOT.jar`.
+
+* Step 2: Click on the `Jobs` tabs at project menu and follow the instructions from the **Jobs** section. Create a new job for the Producer. Select `Spark` as job type and specify the jar file that you just uploaded. The name of the main class is `io.hops.examples.spark.kafka.StreamingExample` and argument is `producer`. At the `Configure and create` tab, click on `Kafka` Services and select the Kafka topic you created at Step 4. Your job page should look like the following:
+
+.. _kafka-producer.png: ../../_images/kafka-producer.png
+.. figure:: ../../imgs/kafka-producer.png
+    :alt: Kafka producer job
+    :target: `kafka-producer.png`_
+    :align: center
+    :figclass: align-center
+
+    Kafka producer job
+
+* Step 3: We repeat the instructions on Step 6 for the Consumer job. Type a different job name and as argument to the main class
+  pass `consumer /Projects/YOUR_PROJECT_NAME/Resources/Data`. The rest
+  remain the same as the Producer job.
+
+**Run the created producer/consumer jobs**
+`Run` both jobs. While the consumer is running you can check its execution log. Use the Dataset browser to navigate to the directory `/Resources/Data-APPLICATION_ID/`. Right click on the file `part-00000` and *Preview* the content.
+
+A sample output would look like the following:
+
+.. _kafka-sink.png: ../../_images/kafka-sink.png
+.. figure:: ../../imgs/kafka-sink.png
+    :alt: Kafka ouput
+    :target: `kafka-sink.png`_
+    :align: center
+    :figclass: align-center
+
+    Kafka output
+
+.. _here: https://github.com/hopshadoop/hops-kafka-examples/tree/master/spark
+.. _HopsUtil: https://github.com/hopshadoop/hops-util
+.. _hops-util-py: https://github.com/logicalclocks/hops-util-py
+
+Example Python Notebook with Kafka Producer and Consumer
+-----------------------
+
+You can find several example notebooks using kafka at hops_examples_.
+
+In this section we will demonstrate how you can use a jupyter notebook and python to produce/consume kafka messages. In this section it is assumed that you have already created a Kafka topic named "test" to produce/consume from and that you have enabled anaconda and installed the python package `kafka-confluent` in your project anaconda environment.
```

Comment:
@Limmen @robzor92 why don't we have the `kafka-confluent` package in the pre-installed python libs?

Reply:
We can pre-install it. Our docs illustrate how to use spark or kafka-confluent for writing Kafka consumers/producers, but you could potentially use any Kafka client, the library is not dependent on it. However, I agree it can be useful to have it pre-installed, since then they can run all hops-examples notebooks out of the box. What do you think? @robzor92

Reply:
Yep let's pre-install